charts/dependency-track/values.schema.json

```diff
@@ -59,6 +59,11 @@
     "apiServer": {
       "type": "object",
       "properties": {
+        "deploymentType": {
+          "type": "string",
```

Comment:
You don't have to provide this for enums.

Reply:
fixed in [ab9b2f9](https://github.com/DependencyTrack/helm-charts/pull/102/commits/ab9b2f91de38ab822fe741df782fcbb32a4b0d3b)